src/Messages.jl

```diff
@@ -74,6 +75,7 @@ import ..bytes
 
 include("ascii.jl")
 
+const nobody = UInt8[]
```

Comment:
Uppercase to match style in other files?

Reply:
this is how its been defined for ages; but yeah, we should probably go through and update it.